### PR TITLE
Editorial: Rename max_batch_lifetime

### PR DESCRIPTION
**Pull request by wangshan**

### CODE REVIEW

**Review by cjpatton — APPROVED** [review]
This seems useful, since it disambiguates "task_lifetime" and "batch_lifetime". The former, as i understand it, is a timestamp; and the latter is a number of queries.

cc/ @tgeoghegan for thoughtful review here

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -443,7 +443,7 @@ in the "type" field (within the DAP URN namespace
 | reportTooEarly             | Report could not be processed because its timestamp is too far in the future. |
 | batchInvalid               | A collect or aggregate-share request was made with invalid batch parameters. |
 | invalidBatchSize           | There are an invalid number of reports in the batch. |
-| batchLifetimeExceeded      | The batch lifetime has been exceeded for one or more reports included in the batch. |
+| batchLifetimeExceeded      | The maximum number of batch queries has been exceeded for one or more reports included in the batch. |
```
Comment (tgeoghegan) [inline draft-ietf-ppm-dap.md:446]: This error type should be renamed to no longer reference "lifetime"

**Review by tgeoghegan — CHANGES_REQUESTED** [review]

**Comment by wangshan** [comment]
@tgeoghegan please have another look

**Review by tgeoghegan — APPROVED** [review]
👍🏻

**Pull request merged**